pkg/internal/webhook/tls/tls.go

```diff
@@ -98,7 +98,6 @@ func New(ctx context.Context, opts Options) (*TLS, error) {
 			SecretNamespace: opts.CASecretNamespace,
 			SecretName:      "cert-manager-approver-policy-tls",
 			RESTConfig:      opts.RestConfig,
-			Log:             log.WithName("certificate-authority"),
```

Comment:
The cert-manager webhook has been refactored. 
Log is now `log` and is private. It is now extracted from the context in the Run method.
 * https://github.com/jetstack/cert-manager/pull/4675